src/metabase/actions/execution.clj

```diff
@@ -203,22 +203,41 @@
       (execute-custom-action! action request-parameters)
       (throw (ex-info (tru "Unknown action type {0}." (name (:type action))) action)))))
 
+(defn- execute-table-action!
+  [kind table-id request-parameters]
+  (let [args
+        {:type     :query
+         :query    {:source-table table-id}
```

Comment:
I know the schema used to require these two weird model keys, but I think they can be removed now.

Reply:
c7112a6e0271